Orchard1.ruleset

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Add a quick note about this at the end of https://github.com/Lombiq/.NET-Analyzers/blob/dev/Docs/AddingAnalyzers.md#how-to-add-the-analyzers-to-your-project-repository as well, please, similar to how the note on General.ruleset is there.

Reply:
I will update the documentation once done.